src/components/ImageSocialShare.vue

```diff
@@ -31,6 +32,9 @@ export default {
     imageURL() {
       return this.image.foreign_landing_url;
     },
+    image_URL() {
```

Comment:
Calling this new computed prop "image_URL" will probably create confusion with the previously existing "imageURL". Please rename "image_URL" to "imageURL" to keep with the camelcase convention, and change the name of "imageURL" to reflect that it returns the foreign landing url instead of the image url.